wren-ui/src/apollo/server/mdl/mdlBuilder.ts

```diff
@@ -206,8 +214,8 @@ export class MDLBuilder implements IMDLBuilder {
         model.columns.push({
           name: column.referenceName,
           type: column.type,
-          isCalculated: column.isCalculated,
-          notNull: column.notNull,
+          isCalculated: column.isCalculated ? true : false,
```

Comment:
How about `column.isCalculated || false`? if you just want to check undefined and null.